src/ai/backend/manager/models/vfolder.py

```diff
@@ -204,6 +204,18 @@ class VFolderOperationStatus(enum.StrEnum):
     DELETE_COMPLETE = "delete-complete"  # vfolder is deleted permanently, only DB row remains
     DELETE_ERROR = "delete-error"
 
+    def is_deletable(self, force: bool = False) -> bool:
+        if force:
+            return self in {
+                VFolderOperationStatus.READY,
+                VFolderOperationStatus.DELETE_PENDING,
+            }
+        else:
+            return self == VFolderOperationStatus.DELETE_PENDING
+
+    def attach_timestamp(self) -> bool:
+        return self == VFolderOperationStatus.DELETE_ONGOING
```

Comment:
What does the name attach_timestamp mean?

Reply:
maybe `timestamp_attachable`?

Reply:
What I meant to ask is how checking `DELETE_ONGOING` is related to `attach_timestamp` function name.

Reply:
I have reverted this change since the name of function is confusing